at-quannguyen/W5/20180718020733_desc_table_user.rb

```diff
@@ -0,0 +1,19 @@
+class DescTableUser < ActiveRecord::Migration[5.1]
+  def change
+    remove_index :users, :email
+    users = User.all
+    users_desc = User.order(id: :desc)
+    users.each_with_index do |elemt, index|
+      obj = users_desc[index]
+      elemt.update_attributes(email: obj.email,
+                              password: obj.password,
+                              team_id: obj.team_id,
+                              single: obj.single,
+                              favorite_song: obj.favorite_song,
+                              gender: obj.gender)
+      team = Team.find_by(user_id: obj.id)
+      team.update_attribute(:user_id, elemt.id) unless team.nil?
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/7b01ff6d5f618841ca8c79e38047e5229d6d21f8/at-quannguyen/W5/20180718020733_desc_table_user.rb#L15"></span>Rails/SkipsModelValidations: Avoid using <code>update_attribute</code> because it skips validations. (http://guides.rubyonrails.org/active_record_validations.html#skipping-validations)</td>
    </tr>
  </tbody>
</table>